### PR TITLE
curl: update to 7.74.0

### PR DESCRIPTION
**Pull request by Astrum-polaris**
I disable the signature checks temporarily because of CI errors
It can be restored later if wanted

### CODE REVIEW

**Comment by lazka** [comment]
> I disable the signature checks temporarily because of CI errors

It should work, please try restoring it.

File: mingw-w64-curl/PKGBUILD [hunk 1]
```diff
@@ -13,13 +13,13 @@ fi
...
+pkgver=7.74.0
 pkgrel=1
 pkgdesc="Command line tool and library for transferring data with URLs. (mingw-w64)"
 arch=('any')
 url="https://curl.haxx.se/"
 license=("MIT")
-makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkgconf")
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-c-ares"
```
Comment (lazka) [inline mingw-w64-curl/PKGBUILD:24]: Please add a dependency on `"${MINGW_PACKAGE_PREFIX}-zstd"`

Comment (Astrum-polaris) [reply mingw-w64-curl/PKGBUILD:25]: Done

**Comment by lazka** [comment]
Thanks!

**Comment by lazka** [comment]
Thanks!

**Pull request merged**